src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,66 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SPLIT_CHARACTER = ",";
+    private static final int WORD_INDEX = 0;
+    private static final int COUNT_INDEX = 1;
+    private static String FROM_FILE_NAME;
+    private static String TO_FILE_NAME;
+    private StringBuilder builder = new StringBuilder();
```

Comment:
this shouldn't be a field

Reply:
but up there Kirill Koval said to make file names as class-level String constants... or i doesn't really understood you?